cluster-scope/base/namespaces/metallb/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: metallb-system
```

Comment:
If the namespace should be named `metallb-system` than the folder shall be named the same. Or the namespace should be `metallb`, then the resource name has to change.